src/extensions/sessionrecording.js

```diff
@@ -9,7 +9,7 @@ const BASE_ENDPOINT = '/e/'
 export const FULL_SNAPSHOT_EVENT_TYPE = 2
 export const META_EVENT_TYPE = 4
 export const INCREMENTAL_SNAPSHOT_EVENT_TYPE = 3
-export const MUTATION_SOURCE_TYPE = 3
+export const MUTATION_SOURCE_TYPE = 0
```

Comment:
This is a bug fix. This should've always been a 0 to match rr-web's types. Somewhat concerning that it was wrong before...